setup.sh

```diff
@@ -1,3 +1,20 @@
 #!/bin/bash
 
 # Add your code here.
+if [ -f meteor.sh ]
+then
+	sh meteor.sh
+else
+	curl https://install.meteor.com/ | sh
+fi
+export PATH=$PATH:$HOME/.meteor
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.0/install.sh | bash
+export NVM_DIR=$HOME/.nvm
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"
+nvm install node
```

Comment:
`nvm install node` for the first time will automatically activate it so you don't have to explicitly run `nvm use node`. However, even with `nvm use node`, it only changes the node version for the current shell session. Other shell sessions will still use the default node.